From 667e9f9ff43accc34af743db028237fd06050f7a Mon Sep 17 00:00:00 2001
From: Albert Chen <R65187@freescale.com>
Date: Fri, 25 Jul 2008 10:53:31 +0800
Subject: [PATCH] ENGR00085139: fixed the wrong USB transcevier type.

Mx31-3ds: when use the default configuration compile, can't pass to USB test.
mx31-3ds use ISP1504 transceiver, but default select MC13783 transceiver,
so it is wrong.

Signed-off-by: Albert Chen <r65187@freescale.com>
---
 arch/arm/configs/imx31_3stack_defconfig |   20 ++++++++++++--------
 1 files changed, 12 insertions(+), 8 deletions(-)

diff --git a/arch/arm/configs/imx31_3stack_defconfig b/arch/arm/configs/imx31_3stack_defconfig
index cf5f731..d13b95a 100644
--- a/arch/arm/configs/imx31_3stack_defconfig
+++ b/arch/arm/configs/imx31_3stack_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.24
-# Tue Jul 22 17:35:54 2008
+# Thu Jul 24 14:50:39 2008
 #
 CONFIG_ARM=y
 CONFIG_SYS_SUPPORTS_APM_EMULATION=y
@@ -184,8 +184,8 @@ CONFIG_ARCH_MXC_HAS_NFC_V2=y
 # CONFIG_I2C_MXC_SELECT3 is not set
 CONFIG_ARCH_HAS_EVTMON=y
 CONFIG_DMA_ZONE_SIZE=24
-CONFIG_MC13783_MXC=y
-CONFIG_MXC_USB_SU6=y
+CONFIG_ISP1504_MXC=y
+# CONFIG_MXC_USB_SU6 is not set
 # CONFIG_MXC_USB_SB3 is not set
 # CONFIG_MXC_USB_DU6 is not set
 # CONFIG_MXC_USB_DB4 is not set
@@ -1143,9 +1143,9 @@ CONFIG_USB_EHCI_ARC=y
 CONFIG_USB_EHCI_ARC_H2=y
 CONFIG_USB_EHCI_ARC_OTG=y
 # CONFIG_USB_STATIC_IRAM is not set
-CONFIG_USB_EHCI_FSL_MC13783=y
+# CONFIG_USB_EHCI_FSL_MC13783 is not set
 # CONFIG_USB_EHCI_FSL_1301 is not set
-# CONFIG_USB_EHCI_FSL_1504 is not set
+CONFIG_USB_EHCI_FSL_1504=y
 # CONFIG_USB_EHCI_FSL_UTMI is not set
 # CONFIG_USB_EHCI_SPLIT_ISO is not set
 CONFIG_USB_EHCI_ROOT_HUB_TT=y
@@ -1250,11 +1250,11 @@ CONFIG_USB_ARC=m
 # CONFIG_USB_GADGET_S3C2410 is not set
 # CONFIG_USB_GADGET_AT91 is not set
 # CONFIG_USB_GADGET_DUMMY_HCD is not set
-# CONFIG_USB_GADGET_DUALSPEED is not set
+CONFIG_USB_GADGET_DUALSPEED=y
 CONFIG_USB_GADGET_ARC_OTG=y
-CONFIG_USB_GADGET_FSL_MC13783=y
+# CONFIG_USB_GADGET_FSL_MC13783 is not set
 # CONFIG_USB_GADGET_FSL_1301 is not set
-# CONFIG_USB_GADGET_FSL_1504 is not set
+CONFIG_USB_GADGET_FSL_1504=y
 # CONFIG_USB_GADGET_FSL_UTMI is not set
 # CONFIG_USB_ZERO is not set
 CONFIG_USB_ETH=m
@@ -1435,6 +1435,10 @@ CONFIG_MXC_BLUETOOTH=m
 CONFIG_GPS_IOCTRL=m
 
 #
+# MXC Media Local Bus Driver
+#
+
+#
 # File systems
 #
 CONFIG_EXT2_FS=y
-- 
1.5.4.4

